### PR TITLE
Feature/348 create object for analysis input with documentation

### PR DESCRIPTION
**Pull request by ArdtK**

### CODE REVIEW

File: ra2ce/analysis/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Analysis
+
+This module contains all protocols and classes related to performing an analysis on a graph or network.
+Analyses can be identified as:
+- `direct`: calculating the **damages** to the infrastructure (roads) due to a hazard (e.g. flood),
+- `indirect`: calculating the economical **losses** that are a consequence of the damage to the infrastructure.
+
+## Input and output
+An analysis consumes an `AnalysisInputWrapper`, containing analysis parameters, the graph/network and some additional settings.
+The output of an analysis is an `AnalysisResultWrapper`, containing the analysis result (`GeoDataFrame`) and again the analysis parameters.
```
Comment (Carsopre) [inline ra2ce/analysis/README.md:10]: I would reformulate this given that the `AnalysisResultWrapper` is only created through an `AnalysisRunner`.

File: ra2ce/analysis/indirect/multi_link_losses.py [hunk 2]
```diff
@@ -8,37 +8,33 @@
...
+from ra2ce.analysis.analysis_input_wrapper import AnalysisInputWrapper
 from ra2ce.analysis.indirect.analysis_indirect_protocol import AnalysisIndirectProtocol
 from ra2ce.analysis.indirect.multi_link_redundancy import MultiLinkRedundancy
 from ra2ce.network.graph_files.graph_file import GraphFile
 from ra2ce.network.hazard.hazard_names import HazardNames
 
 
 class MultiLinkLosses(AnalysisIndirectProtocol):
-    graph_file: GraphFile
+    analysis_input: AnalysisInputWrapper
```
Comment (Carsopre) [inline ra2ce/analysis/indirect/multi_link_losses.py:19]: I do not think you should be keeping this property here. It should be consumed during the `__init__` therefore its contents "unwrapped" into explicit properties for the class.

File: ra2ce/analysis/indirect/multi_link_origin_destination.py [hunk 3]
```diff
@@ -21,33 +22,27 @@
 
 
 class MultiLinkOriginDestination(AnalysisIndirectProtocol):
-    graph_file: GraphFile
+    analysis_input: AnalysisInputWrapper
```
Comment (Carsopre) [inline ra2ce/analysis/indirect/multi_link_origin_destination.py:25]: Is it really needed?

**Review by Carsopre — CHANGES_REQUESTED** [review]

**Review by Carsopre — APPROVED** [review]

File: ra2ce/analysis/indirect/multi_link_origin_destination.py (hunk 3 above)
Comment (ArdtK) [reply ra2ce/analysis/indirect/multi_link_origin_destination.py:25]: It is passed to a next analysis. Made private.

File: ra2ce/analysis/indirect/multi_link_losses.py (hunk 2 above)
Comment (ArdtK) [reply ra2ce/analysis/indirect/multi_link_losses.py:19]: Made private.

File: ra2ce/analysis/README.md (hunk 1 above)
Comment (ArdtK) [reply ra2ce/analysis/README.md]: Done

**Pull request merged**